code/game/machinery/kitchen/cooking_machines/oven.dm

```diff
@@ -50,24 +50,38 @@
 
 
 /obj/machinery/appliance/cooker/oven/AltClick(var/mob/user)
+	.=1
 	if(user.stat || user.restrained())	return
 	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)//No spamming the door, it makes a sound
 	toggle_door()
 
+
 /obj/machinery/appliance/cooker/oven/verb/toggle_door()
-	set src in view()
+	set src in world //So that AI can operate it remotely
```

Comment:
I dont like this for the following reasons:
AIs can use alt-click
AIs can use rightclick -> Open / Close oven door
What if we decide to add more than one oven at a later point ?

Therefore I think AIs can live without having the Open/Close oven door verb in their object tab